src/cairoWriter/writers/cairoImportFunctionDefinitionWriter.ts

```diff
@@ -5,6 +5,10 @@ import { CairoASTNodeWriter } from '../base';
 // Not being used as for now
 export class CairoImportFunctionDefinitionWriter extends CairoASTNodeWriter {
   writeInner(node: CairoImportFunctionDefinition, _writer: ASTWriter): SrcDesc {
+    if (node.path[node.path.length - 1].endsWith('Trait')) {
+      return [`use ${node.path.join('::')};`];
+    }
```

Comment:
Why is it needed?

Reply:
Functions inside traits are not imported directly. The path is cut short at the name. Without the condition we would import `import::path::Trait::funcName` which is wrong. We should only use  `import::path::Trait`.

Reply:
Oh I see, let's explain it in the comment, I suppose it's only a temporary fix